mdtraj/io.py

```diff
@@ -72,11 +75,13 @@
 ---------
 """
 from __future__ import print_function, division
```

Comment:
this also needs `import io`, and probably `from __future__ import absolute_import`.